src/main/java/com/ibm/misc/Debug.java

```diff
@@ -194,7 +195,7 @@ public Object run() {
      *   <code>TYPE_ENTRY_EXIT</code> has been split into
      *   <code>TYPE_ENTRY</code> and <code>TYPE_EXIT</code>.
      */
-
+    @Deprecated
```

Comment:
Why deprecate this field?

Reply:
If you check the comment above, this field is supposed to be deprecated. It just seems like we never declared it.

Reply:
OK i see that now.